content/en/tracing/profiling/_index.md

```diff
@@ -194,6 +201,20 @@ Once enabled, the following profile types are collected:
 | Socket I/O               | Shows the time each method spent handling socket I/O.                                                                                                                                                                                                                                              |
 
 
+{{% /tab %}}
+
+{{% tab "Python" %}}
+
+Once enabled, the following profile types are collected:
+
+| Profile type             | Definition                                                                                                                                                                                                                                                                                         |
+| ------------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| CPU         | Shows the time each method spent running on the CPU. It includes CPython bytecode, but not native code called from within Python.                                                                                                                                                                     |
+| Allocation               | Shows the amount of heap memory allocated by each method, including allocations which were subsequently freed.                                                                                                                                                                                     |
+| Wall | Shows the elapsed time each method spent. Elapsed time includes time when code is running on CPU, waiting for I/O, and anything else that happens while the method is running. |
+| Exceptions               | Shows the number of exceptions caught or uncaught for each method.                                                                                                                                                                                                                                                 |
```

Comment:
@jd The method is where the exception was raised?

Reply:
Yes.